metricbeat/docs/fields.asciidoc

```diff
@@ -6890,12 +6890,12 @@ Kubernetes pod name
 
 --
 
-*`kubernetes.pod.uid`*::
+*`kubernetes.uid`*::
```

Comment:
Breaking change? should we keep both fields till 7.0?

Reply:
`kubernetes.pod.uid` is still unreleased, so this change should be ok

Reply:
Oh ok :)

Reply:
Oh, it's not released yet, then my comment above may apply :-)

Reply:
For my education: This is unique for 1 kubernetes "cluster". Is there also such a thing as a unique pod it?

Reply:
Can there be cases where more then one of the id's from 2 different resources is in the same event? For example having the `pod.uid` and the `kubernetes.uid` which describes the overall cluster in one event? If yes, I would probably put each `uid` under it's resource. Also correlation wise it probably only makes sense to correlate `pod.uid` but not correlated it with any other resources?

Reply:
That case can happen, since https://github.com/elastic/beats/pull/7231

But same would apply to labels/annotations. There is always a main resource for the event, and we put the labels & annotations for that one. Would it make sense to move those under `pod.labels`?

Reply:
I so far assume `labels` is a more generic thing and similar labels apply to different resources. A query could be "give me all resources with label `a`". Based on this assumption I think the current handling of labels and tags make still sense. But if the labels don't overlap like in the case of the `uid` the above would seem to make more sense.

Reply:
Pushed b6d8f7628 to move it back to `pod.uid`